typing/typedecl.ml

```diff
@@ -935,7 +935,7 @@ let compute_variance_type env check (required, loc) decl tyl =
 
 let add_false = List.map (fun ty -> false, ty)
 
-(* A parameter is constrained if either is is instantiated,
+(* A parameter is constrained if either it is instantiated,
```

Comment:
`if either` is not idiomatic: "is constrained either if..." or, better to me at least, "is constrained if it is either instantiated, or is a variable..."